### PR TITLE
Fix theme panel `undefined` key errors in dev

### PR DESCRIPTION
**Pull request by vladmoroz**

### CODE REVIEW

File: packages/radix-ui-themes/src/components/theme-panel.tsx [hunk 1]
```diff
@@ -135,7 +135,7 @@ const ThemePanelImpl = React.forwardRef<ThemePanelImplElement, ThemePanelImplPro
       function handleKeydown(event: KeyboardEvent) {
         const isModifierActive = event.altKey || event.ctrlKey || event.shiftKey || event.metaKey;
         const isKeyboardInputActive = document.activeElement?.closest(keyboardInputElement);
-        const isKeyT = event.key.toUpperCase() === 'T' && !isModifierActive;
+        const isKeyT = event.key?.toUpperCase() === 'T' && !isModifierActive;
```
Comment (vladmoroz) [inline packages/radix-ui-themes/src/components/theme-panel.tsx:138]: TS doesn't think `key` could be undefined, but I'm definitely seeing keydown events without `key` property when using keyboard early on page load

Comment (benoitgrelard) [reply packages/radix-ui-themes/src/components/theme-panel.tsx:138]: wow, weird, I didn't think this was possible… We never check this in primitives, don't feel like that should be needed…

**Review by benoitgrelard — APPROVED** [review]

File: packages/radix-ui-themes/src/components/theme-panel.tsx (hunk 1 above)
Comment (vladmoroz) [reply packages/radix-ui-themes/src/components/theme-panel.tsx:138]: Was also surprised. I get this on the first keypress in sink
image

Comment (benoitgrelard) [reply packages/radix-ui-themes/src/components/theme-panel.tsx:138]: weird, I don't see this.

**Pull request merged**